bongo/lang/en/bongo.php

```diff
@@ -41,6 +41,8 @@
 $string['bongoinfo'] = 'For more information on Bongo and how it works see: <a href="http://www.bongolearn.com">www.bongolearn.com</a>';
 $string['plugindescription'] = 'Bongo';
 $string['pluginname'] = 'Bongo';
+$string['modulename'] = 'Bongo';
```

Comment:
Because we are in the activity modules section, we are required to have two additional strings, the module name and the plural module name